### ISSUE TITLE
bug report after running several steps

### ISSUE THREAD

**Issue by leekum2018** [ltzheng/SimpleTIR#11]
After running 15 steps, a bug comes out with the follow  error.

<img width="2338" height="1190" alt="Image" src="https://github.com/user-attachments/assets/4bb6574a-cbf4-445e-bc51-1b74a2659a61" />

I have not try with default settings, since this will encounter OOM. So I just adjusted some training hyper parameter, like PPO_MINI_BATCH_SIZE, PPO_MICRO_TOKEN, to accommodate my gpu. And this error appears. How to fix this bug?

Thanks!

**Comment by AIDefender** [comment]
Hi, thanks for reaching out! Can you provide your hyperparameters or running script, so that we can reproduce this bug? I remember that I also encounter this bug during development, with this raw key id. I believe this has something to do with the dynamic sampling module, i.e., select samples with accuracy 0.1~0.9. I guess if the batch size is small, it may lead to an empty group when computing advantage. In such case, remove `acc_filter` in command line args may help.

**Comment by leekum2018** [comment]
@AIDefender 
My running script is as follows

`bash train.sh \
  --max_response_length 8000 \
  --max_prompt_length 16000 \
  --rollout_tp 2 \
  --save_freq -1 \
  --model_name DeepSeek-R1-Distill-Qwen-7B \
  --max_turns 5 \
  --train_dataset "simplelr_math_35/train deepscaler/train" \
  --acc_filter 0.1_0.9 \
  --remove_clip True \
  --mask_void_turns True \
  --oversample 3
`

**Comment by AIDefender** [comment]
Thanks for replying. This may happen when all responses from a prompt are masked, due to overlong filtering. We recommend longer response length or setting remove_clip to False when using distill models. This may help solving the issue. Also, we did not test our codebase on distill models. Training with Qwen Base models should be ok.
